core/io/file_access_compressed.cpp

```diff
@@ -125,10 +125,11 @@ Error FileAccessCompressed::_open(const String &p_path, int p_mode_flags) {
 		char rmagic[5];
 		f->get_buffer((uint8_t *)rmagic, 4);
 		rmagic[4] = 0;
-		if (magic != rmagic || open_after_magic(f) != OK) {
+		Error err = ERR_FILE_UNRECOGNIZED;
```

Comment:
Shadows previous `err`:
```suggestion
		err = ERR_FILE_UNRECOGNIZED;
```

Reply:
Fixed :+1: